app/controllers/posts_controller.rb

```diff
@@ -1,4 +1,5 @@
 class PostsController < ApplicationController
+  # load_and_authorize_resource
```

Comment:
Great job team 🤝

- [ ] [Optional] Kindly make sure you use CanCanCan for Post authorization in Post Controller.